sway/criteria.c

```diff
@@ -35,18 +35,19 @@ bool criteria_is_empty(struct criteria *criteria) {
 }
 
 void criteria_destroy(struct criteria *criteria) {
+	free(criteria->raw);
+	free(criteria->cmdlist);
+	free(criteria->target);
```

Comment:
I don't think you want to free the target. I'll look into this later if no one has confirmed this.

Reply:
It's a `char*`, are you suggesting that something else might use the pointer?

Reply:
Oh, right. Freeing it is fine then.